generated/grpc/Gsuite/V1/GSuitePartnerClient.php

```diff
@@ -0,0 +1,62 @@
+<?php
+// GENERATED CODE -- DO NOT EDIT!
+
+namespace Gsuite\V1;
+
+/**
+ * GSuitePartner provides RPCs to create and poll G Suite accounts
+ */
+class GSuitePartnerClient extends \Grpc\BaseStub {
+
+    /**
+     * @param string $hostname hostname
+     * @param array $opts channel options
+     * @param \Grpc\Channel $channel (optional) re-use channel object
+     */
+    public function __construct($hostname, $opts, $channel = null) {
+        parent::__construct($hostname, $opts, $channel);
+    }
+
+    /**
+     * Checks if a domain is available, returns AlreadyExists (409 Conflict) if the domain is not available
+     * @param \Gsuite\V1\GetDomainAvailabilityRequest $argument input argument
+     * @param array $metadata metadata
+     * @param array $options call options
+     */
+    public function GetDomainAvailability(\Gsuite\V1\GetDomainAvailabilityRequest $argument,
+      $metadata = [], $options = []) {
+        return $this->_simpleRequest('/gsuite.v1.GSuitePartner/GetDomainAvailability',
+        $argument,
+        ['\Google\Protobuf\Empty', 'decode'],
+        $metadata, $options);
+    }
+
+    /**
+     * Checks if the domain is verified for a G Suite account
```

Comment:
Doesn't it verify ownership of the domain, not check if it is verified

Reply:
Oh, I'm guessing this is just an unfortunate code comment from our grpc repo, and so it made it here during sdk generation

Reply:
Yeah, those two RPCs already existed.